### DISCUSSION TITLE
Abaqus .inp file to .stl file

### DISCUSSION THREAD

**Discussion by RBehera87** [nschloe/meshio#1146] [Q&A]
Hi,

I have a Abaqus .inp file. Can I convert this .inp file to .stl file using meshio?   
I'm trying to read it using meshio.read() after installing meshio[all]

import meshio

mesh = meshio.read(data.inp)
Error: AttributeError: module 'meshio' has no attribute 'read'

Any help?

**Comment by nschloe** [comment]
Looks like a broken installation. What's your meshio version?
